lib/archethic_web/templates/dir_listing/index.html.heex

```diff
@@ -147,6 +148,7 @@
                             </tbody>
                         </table>
                     </td>
+                    <td class="size"><%= file.size %></td>
```

Comment:
same "size" css class isn't defined

Reply:
@tenmoves 

Yes I know, but i am asking you how to view this indexof screen in localhost.

Reply:
also file.size will not work because we don't provide this field in lib/archethic_web/controllers/api/web_hosting_controller/directory_listing.ex